Assets/AnkrSDK/Runtime/Data/ABIStringLoader.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.IO;
+using UnityEngine;
+
+namespace AnkrSDK.Data
+{
+	public class ABIStringLoader
+	{
+		private readonly string _pathToAbiFolder;
+
+		public ABIStringLoader(string relativePath)
+		{
+			_pathToAbiFolder = Path.Combine(Application.dataPath, relativePath);
```

Comment:
there is no reason to create an instance of that class. Let's make it a static helper.